tests/positive/general/general.go

```diff
@@ -706,3 +738,72 @@ func EmptyUserdata() types.Test {
 		ConfigShouldBeBad: true,
 	}
 }
+
+// VerifyMergingOfMultipleBaseConfigs checks if multiple
+// base configs and/or platform configs are merged in the
+// right order.
```

Comment:
It doesn't actually test ordering because each config is disjoint.  Let's additionally set up some different file contents at the same paths to ensure later config filenames override earlier ones and platform overrides base.